src/components/layout/nav/models/nav-types.ts

```diff
@@ -0,0 +1,10 @@
+export type NavTypes = {
+    base?: string
```

Comment:
Why is not the types fixed?

Reply:
these properties are the same in different nav component, because of that I use global types for them, but when I checked layout we didn't use all the properties so I did that, but I'll remove the extra property in my next changes